README.md

```diff
@@ -27,22 +27,23 @@ http://help.thisdata.com
 
 #### Configuration
 
-Configure ThisData as follows:
+An example ThisData configuration is below. See `this_data/configuration.rb` for
+more options. For example, in production you will probably want asynchronous and
```

Comment:
Both asynchronous ([`track_with_response`](https://github.com/thisdata/thisdata-ruby/blob/108ff10031d3c8ea66fbf3fc1611528d4762388b/lib/this_data.rb#L84-L91)) and synchronous ([`track_async`](https://github.com/thisdata/thisdata-ruby/blob/108ff10031d3c8ea66fbf3fc1611528d4762388b/lib/this_data.rb#L99-L102)) have rescues, as it should be enforced that _any_ issue in our API will _never_ cause bad consequences. If a user could mess up their app by changing / not changing a not-obvious config setting, that'd be bad.

Is that sufficient?